.github/workflows/cd-pipeline.yml

```diff
@@ -116,4 +116,22 @@ jobs:
     with:
       environment: prod
       ref: ${{ needs.create-and-post-tag.outputs.newVersion }}
-      lambdaDeploy: true
\ No newline at end of file
+      lambdaDeploy: true
+
+  send-slack-notifications:
+    needs: [deploy-to-prod, create-and-post-tag]
```

Comment:
since deploy to prod requires create-and-post-tag to run, do we really need to specify this again?